src/parse.h

```diff
@@ -237,6 +237,8 @@ struct net_definition {
     gboolean optional;
     NetplanOptionalAddressFlag optional_addresses;
     gboolean critical;
+    /* netplan-feature: activation-mode */
+    char* activation_mode;
```

Comment:
Please move this to the very end of the `net_definition` struct, otherwise we might break the ABI compatibility (again).

Reply:
Yes! Done!